### PR TITLE
Reference redirect

### PR DESCRIPTION
**Pull request by akmiller01**
Captures all urls that begin with a version number and redirects them to reference.iatistandard.org

### CODE REVIEW

File: iati/iati/urls.py [hunk 1]
```diff
@@ -23,6 +25,10 @@
     url(r'^documents/', include(wagtaildocs_urls)),
 ]
 
+urlpatterns += [
+    url(r'^(104|105|201|202|203)/', reference_redirect)
```
Comment (dalepotter) [inline iati/iati/urls.py]: On reflection, I would add `101|102|103` to this, given I've just checked the outcome of those URLs

- http://iatistandard.org/101
- http://iatistandard.org/102
- http://iatistandard.org/103

File: iati/home/views.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+from django.shortcuts import redirect
+
+
+def reference_redirect(request, *args, **kwargs):
+    """A functional view that accepts any request starting with a version number."""
+    base_url = "http://reference.iatistandard.org"
+    slug = request.get_full_path()
+    redirection_url = base_url + slug
```
Comment (dalepotter) [inline iati/home/views.py]: I'm guessing `slug` will always include a leading slash?

Comment (akmiller01) [reply iati/home/views.py]: Yes.

File: iati/iati/urls.py (hunk 1 above)
Comment (akmiller01) [reply iati/iati/urls.py]: :+1:

**Pull request merged**